flutter/macos/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -0,0 +1 @@
+../../ios/Classes/SwiftSentryFlutterPlugin.swift
```

Comment:
I symlinked the iOS file to the MacOS folder.

Reply:
that's fancy :)